test/brod_producer_SUITE.erl

```diff
@@ -34,16 +34,20 @@
 
 %% Test cases
 -export([ t_produce_sync/1
+        , t_produce_sync_offset/1
         , t_produce_no_ack/1
+        , t_produce_no_ack_offset/1
         , t_produce_async/1
         , t_producer_topic_not_found/1
         , t_producer_partition_not_found/1
         , t_produce_partitioner/1
         , t_produce_batch/1
+        , t_produce_buffered_offset/1
         ]).
 
 -include_lib("common_test/include/ct.hrl").
 -include_lib("eunit/include/eunit.hrl").
+-include("brod.hrl").
```

Comment:
I noticed today that this is redundant. Should I remove it?

Reply:
yes please.